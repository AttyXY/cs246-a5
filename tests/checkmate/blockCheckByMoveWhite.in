setup
+ K e1
+ k e5
+ r a2
+ r b3
+ B h6
= black
done
game human human
move b3 b1
move h6 c1 
move b1 d1
move b1 c1
