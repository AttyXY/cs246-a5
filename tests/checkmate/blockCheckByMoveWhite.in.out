WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

